apps/home/app/@readme/_shared/components/ReadmeBox/index.tsx

```diff
@@ -0,0 +1,5 @@
+import type { PropsWithChildren } from 'react'
+
+export const ReadmeBox = ({ children }: PropsWithChildren) => (
+  <div className="border border-[#d0d7de] rounded-[6px] p-[24px]">{children}</div>
```

Comment:
```suggestion
  <div className="border-slate-200 border-2 rounded-lg p-[24px]">{children}</div>
```

어떠신가요? MBTIOverview 에서 쓰고 있는 보더 설정이라, tw theme 사용하는 것도 방법일 것 같네요

Reply:
왕 좋아요! [89cba79](https://github.com/hamsurang/origin/pull/32/commits/89cba7904ae999f2a577220ffa48255a3dd1da99)에서 반영 하였습니다!